### PR TITLE
Doc update core:int

### PR DESCRIPTION
**Pull request by jpsiitonen**
int class examples added.
Please verify:  >>> ( unsigned right shift operator ) any ideas how to improve example?

### CODE REVIEW

File: sdk/lib/core/int.dart [hunk 1]
```diff
@@ -59,6 +64,11 @@ abstract class int extends num {
   ///
   /// If both operands are negative, the result is negative, otherwise
   /// the result is non-negative.
+  /// ```dart
+  /// print((2 & 1).toRadixString(2)); // 0010 & 0001 -> 0000
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:68]: Dartpad `0`

File: sdk/lib/core/int.dart [hunk 2]
```diff
@@ -59,6 +64,11 @@ abstract class int extends num {
...
+  /// print((3 & 1).toRadixString(2)); // 0011 & 0001 -> 0001
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:69]: Dartpad `1`

File: sdk/lib/core/int.dart [hunk 3]
```diff
@@ -59,6 +64,11 @@ abstract class int extends num {
...
+  /// print((10 & 2).toRadixString(2)); // 1010 & 0010 -> 0010
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:70]: Dartpad `10`

File: sdk/lib/core/int.dart [hunk 4]
```diff
@@ -69,6 +79,13 @@ abstract class int extends num {
   ///
   /// If both operands are non-negative, the result is non-negative,
   /// otherwise the result is negative.
+  ///
+  /// Example:
+  /// ```dart
+  /// print((2 | 1).toRadixString(2)); // 0010 | 0001 -> 0011
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:85]: Dartpad `11`

File: sdk/lib/core/int.dart [hunk 5]
```diff
@@ -69,6 +79,13 @@ abstract class int extends num {
...
+  /// print((3 | 1).toRadixString(2)); // 0011 | 0001 -> 0011
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:86]: Dartpad `11`

File: sdk/lib/core/int.dart [hunk 6]
```diff
@@ -79,6 +96,13 @@ abstract class int extends num {
   ///
   /// If the operands have the same sign, the result is non-negative,
   /// otherwise the result is negative.
+  ///
+  /// Example:
+  /// ```dart
+  /// print((2 ^ 1).toRadixString(2)); //  0010 ^ 0001 -> 0011
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:102]: Dartpad `11`

File: sdk/lib/core/int.dart [hunk 7]
```diff
@@ -79,6 +96,13 @@ abstract class int extends num {
...
+  /// print((3 ^ 1).toRadixString(2)); //  0011 ^ 0001 -> 0010
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:103]: Dartpad `10`

File: sdk/lib/core/int.dart [hunk 8]
```diff
@@ -99,6 +123,13 @@ abstract class int extends num {
   /// mask.
   ///
   /// It is an error if [shiftAmount] is negative.
+  ///
+  /// Example:
+  /// ```dart
+  /// print((3 << 1).toRadixString(2)); // 0011 -> 0110
+  /// print((9 << 2).toRadixString(2)); // 1001 -> 100100
+  /// print((10 << 3).toRadixString(2)); // 1010 -> 01010000
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:131]: Dartpad `1010000`

File: sdk/lib/core/int.dart [hunk 9]
```diff
@@ -99,6 +123,13 @@ abstract class int extends num {
   /// mask.
   ///
   /// It is an error if [shiftAmount] is negative.
+  ///
+  /// Example:
+  /// ```dart
+  /// print((3 << 1).toRadixString(2)); // 0011 -> 0110
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:129]: Dartpad `110`

File: sdk/lib/core/int.dart [hunk 10]
```diff
@@ -108,6 +139,14 @@ abstract class int extends num {
   ///`pow(2, shiftIndex)`.
   ///
   /// It is an error if [shiftAmount] is negative.
+  ///
+  /// Example:
+  ///
+  /// ```dart
+  /// print((3 >> 1).toRadixString(2)); // 0011 -> 0001
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:145]: Dartpad `1`

File: sdk/lib/core/int.dart [hunk 11]
```diff
@@ -108,6 +139,14 @@ abstract class int extends num {
...
+  /// print((9 >> 2).toRadixString(2)); // 1001 -> 0010
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:146]: Dartpad `10`

File: sdk/lib/core/int.dart [hunk 12]
```diff
@@ -108,6 +139,14 @@ abstract class int extends num {
...
+  /// print((10 >> 3).toRadixString(2)); // 1010 -> 0001
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:147]: Dartpad `1`

File: sdk/lib/core/int.dart [hunk 13]
```diff
@@ -117,6 +156,15 @@ abstract class int extends num {
   /// and zero-bits are shifted in as the new most significant bits.
   ///
   /// The [shiftAmount] must be non-negative.
+  ///
+  /// Example:
+  ///
+  /// ```dart
+  /// print((3 >>> 1).toRadixString(2)); // 0011 -> 0001
+  /// print((9 >>> 2).toRadixString(2)); // 1001 -> 0010
+  /// print(((-9) >>> 2).toRadixString(2));
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:163]: I don't know about this example. Anyway for web it is `111111111111111111111111111101`

File: sdk/lib/core/int.dart [hunk 14]
```diff
@@ -117,6 +156,15 @@ abstract class int extends num {
   /// and zero-bits are shifted in as the new most significant bits.
   ///
   /// The [shiftAmount] must be non-negative.
+  ///
+  /// Example:
+  ///
+  /// ```dart
+  /// print((3 >>> 1).toRadixString(2)); // 0011 -> 0001
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:161]: Dartpad `1`

File: sdk/lib/core/int.dart [hunk 15]
```diff
@@ -117,6 +156,15 @@ abstract class int extends num {
...
+  /// print((9 >>> 2).toRadixString(2)); // 1001 -> 0010
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:162]: Dartpad `10`

File: sdk/lib/core/int.dart [hunk 16]
```diff
@@ -279,6 +334,22 @@ abstract class int extends num {
   /// '9', with 'a' being 10 an 'z' being 35.
   ///
   /// The [radix] argument must be an integer in the range 2 to 36.
+  ///
+  /// Example:
+  /// ```dart
+  /// // Binary (base 2).
+  /// print(12.toRadixString(2)); // 1100
+  /// print(31.toRadixString(2)); // 00011111
+  /// print(2021.toRadixString(2)); // 011111100101
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:341]: Dartpad `11111100101`

File: sdk/lib/core/int.dart [hunk 17]
```diff
@@ -279,6 +334,22 @@ abstract class int extends num {
   /// '9', with 'a' being 10 an 'z' being 35.
   ///
   /// The [radix] argument must be an integer in the range 2 to 36.
+  ///
+  /// Example:
+  /// ```dart
+  /// // Binary (base 2).
+  /// print(12.toRadixString(2)); // 1100
+  /// print(31.toRadixString(2)); // 00011111
```
Comment (alexlindroos) [inline sdk/lib/core/int.dart:340]: Dartpad `11111`

**Review by stepthomp — APPROVED** [review]

**Pull request closed without merging**